### PR TITLE
[Doctest] Add configuration_time_series_transformer.py

### PR DESCRIPTION
**Pull request by SD-13**
# What does this PR do?

Fixes part of issue https://github.com/huggingface/transformers/issues/19487.
Adds `configuration_time_series_transformer.py` to `Doc tests`.

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [X] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [X] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by SD-13** [comment]
Hey @ydshieh, I just had a doubt

> Change the import order of the model and configuration classes

here, which order do you mean, the ascending or descending? because in `configuration_time_series_transformer.py`, it was already in ascending order.

File: src/transformers/models/time_series_transformer/configuration_time_series_transformer.py [hunk 1]
```diff
@@ -112,12 +112,12 @@ class TimeSeriesTransformerConfig(PretrainedConfig):
         Example:
 
     ```python
-    >>> from transformers import TimeSeriesTransformerConfig, TimeSeriesTransformerModel
+    >>> from transformers import TimeSeriesTransformerModel, TimeSeriesTransformerConfig
```
Comment (ydshieh) [inline src/transformers/models/time_series_transformer/configuration_time_series_transformer.py:115]: Config should go before Model, which is already the case in the current `main` branch. No need to change here.

File: utils/documentation_tests.txt [hunk 2]
```diff
@@ -95,6 +95,7 @@ src/transformers/models/speech_to_text/modeling_speech_to_text.py
 src/transformers/models/speech_to_text_2/modeling_speech_to_text_2.py
 src/transformers/models/segformer/modeling_tf_segformer.py
 src/transformers/models/swin/modeling_swin.py
+src/transformers/models/time_series_transformer/configuration_time_series_transformer.py
```
Comment (ydshieh) [inline utils/documentation_tests.txt:98]: This order is good @SD-13 💯

File: src/transformers/models/time_series_transformer/configuration_time_series_transformer.py (hunk 1 above)
Comment (SD-13) [reply src/transformers/models/time_series_transformer/configuration_time_series_transformer.py:115]: okay.

File: utils/documentation_tests.txt (hunk 2 above)
Comment (SD-13) [reply utils/documentation_tests.txt:98]: okay.

**Comment by SD-13** [comment]
@ydshieh, By mistake, I pushed some merged changes, I will revert them soon.

**Comment by SD-13** [comment]
@ydshieh PTAL. Thanks,

File: src/transformers/models/time_series_transformer/configuration_time_series_transformer.py (hunk 1 above)
Comment (ydshieh) [reply src/transformers/models/time_series_transformer/configuration_time_series_transformer.py:115]: My comment here  is not addressed yet.

Comment (SD-13) [reply src/transformers/models/time_series_transformer/configuration_time_series_transformer.py]: Sorry, previously I missed it. now it is done

**Review by ydshieh — APPROVED** [review]
A time series of pull request and pull accept! Thanks!

**Comment by SD-13** [comment]
> A time series of pull request and pull accept! Thanks!

That sounds AWESOME!!

**Pull request merged**